.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,105 @@
+name: Bug report
+description: Bugs in Porcupine
+title: "Porcupine Issue: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before** reporting an issue, make sure to read the [documentation](https://picovoice.ai/docs/porcupine/) and search [existing issues](https://github.com/Picovoice/porcupine/issues).
+  - type: checkboxes
+    id: check
+    attributes:
+      label: Have you checked the docs and existing issues?
+      description: Make sure you have checked all of the below before submitting an issue
+      options:
+        - label: I have read all of the relevant Picovoice Porcupine docs
+          required: true
+        - label: I have searched the existing issues for Porcupine
+          required: true
+  - type: dropdown
+    id: sdk
+    attributes:
+      label: SDK
+      options:
```

Comment:
Porcupine and Picovoice need `MCU` listed under SDKs as well. Microcontroller packages are only available for those two products I believe